core/src/main/java/org/apache/gravitino/authorization/OwnerManager.java

```diff
@@ -64,6 +65,9 @@ public OwnerManager(EntityStore store) {
   @Override
   public void setOwner(
       String metalake, MetadataObject metadataObject, String ownerName, Owner.Type ownerType) {
+    // TODO: Support GROUP type owner in the future.
```

Comment:
Can you create an issue and add the issue id here as the TODO comment?

Reply:
Fixed.